src/pgduckdb_duckdb.cpp

```diff
@@ -186,9 +186,23 @@ DuckDBManager::LoadExtensions(duckdb::ClientContext &context) {
 	auto duckdb_extensions = ReadDuckdbExtensions();
 
 	for (auto &extension : duckdb_extensions) {
-		if (extension.enabled) {
-			DuckDBQueryOrThrow(context, "LOAD " + extension.name);
+		if (!extension.enabled) {
+			continue;
 		}
+
+		/*
+		 * Skip the httpfs extension. It conflicts with our cached_httpfs
+		 * extension which is loaded by default, but people might still try to
+		 * install it manually. We could also throw an error by doing so, but
+		 * it seems nicer to just skip it, because it is clear what they meant
+		 * to do and this way we don't need to educate people about the
+		 * existence of our cached_httpfs extension (which might be removed in
+		 * the near future anyway).
+		 */
+		if (extension.name == "httpfs") {
+			continue;
+		}
+		DuckDBQueryOrThrow(context, "LOAD " + extension.name);
```

Comment:
Can we still try to load it for those who can?
```suggestion
		if (extension.name == "httpfs") {
			continue;
		}
		try {
  		DuckDBQueryOrThrow(context, "LOAD " + extension.name);
  	} catch (duckdb::Exception& e) {
  	   if (extension.name != "httpfs") {
  			 throw;
			 }
		}
```